custom/opm/health_status.py

```diff
@@ -187,7 +188,10 @@ class AWCHealthStatus(object):
     # our hearts' content.  This would allow us to override definitions of
     # indicators based on their meanings in THIS report.
     def __init__(self, cases):
-        self.cases = cases
+        # Some of the cases are second or third children of the same mother
+        # include that distinction here
+        self.all_cases = cases
```

Comment:
Shouldn't the report be looking at forms-submitted during the selected month, because a case may be status mother now, but might be status-pregnant four months back if the report month selected is four months back (Was just reviewing since there was a question from OPM). Apologies, if I am interpreting this incorrectly.